api/models/BedEvent.js

```diff
@@ -58,15 +58,10 @@ const model = {
     }
   },
 
-  getOOCByCentreGroupByGenderAndReason: (centreId) =>
-    BedEvent.getCurrentOOCByCentre(centreId)
-      .then(BedEvent.groupByGender)
-      .then(BedEvent.groupAndCountByReason)
-      .then(BedEvent.fillInBlanks),
-
-  fillInBlanks: (inp) => {
+  // centreId param is unused however e2e tests will fail when removed
+  getOOCByCentreGroupByGenderAndReason: (centreId) => {
```

Comment:
could do with a better name, it's not doing any of what the function name suggests